plugins/out_stackdriver/stackdriver.c

```diff
@@ -2994,6 +3006,7 @@ static void cb_stackdriver_flush(struct flb_event_chunk *event_chunk,
 #ifdef FLB_HAVE_METRICS
     if (ret_code == FLB_OK) {
         cmt_counter_inc(ctx->cmt_successful_requests, ts, 1, (char *[]) {name});
+        cmt_histogram_observe(ctx->cmt_write_entries_latency, ts, write_entries_latency, 1, (char *[]) {name});
```

Comment:
Since there is a chance that `write_entries_latency` can be `0.0`, due to the if statement on line 2937, perhaps this should also be guarded by a check that `write_entries_latency` is greater than `0.0`.

Reply:
Done.